### PR TITLE
fix(ci): adjust tag release workflow

### PR DESCRIPTION
**Pull request by inc-man**
# Motivation

The last run failed due to no credentials. We want to change that for future runs.

# Changes

- add credentials
- add token to release_drafter step
- adjust release_drafter template

### CODE REVIEW

File: .github/workflows/bump-version.yml [hunk 1]
```diff
@@ -28,10 +28,10 @@ jobs:
...
       uses: ./.github/actions/prepare
     
-    - name: Set up Git
-      run: |
-        git config user.name "github-actions[bot]"
-        git config user.email "github-actions@github.com"
+#    - name: Set up Git
+#      run: |
+#        git config user.name "github-actions[bot]"
+#        git config user.email "github-actions@github.com"
```
Comment (AntonioVentilii-DFINITY) [inline .github/workflows/bump-version.yml:34]: removing this should be in another PR

Comment (inc-man) [reply .github/workflows/bump-version.yml]: uncommented in this PR

**Review by AntonioVentilii-DFINITY — APPROVED** [review]
